### ISSUE TITLE
Warfarin Module - Mouse Cursor Movement Reset

### ISSUE THREAD

**Issue by antom** [pdjstone/archimedes-live#35]
The ['Warfarin' module](https://stardot.org.uk/forums/viewtopic.php?p=231421&fbclid=IwY2xjawG6iHZleHRuA2FlbQIxMAABHVOUDY9PA73-8NUFHcPsVGKk6Zf-EaRbn3SQHBwvG7lB0j1QII-97XpiiA_aem_08Uqv_yoDhZw5JlyptZaHg#p231421) that allows the mouse cursor to be moved by pressing keys does not work as expected - when pressing the assigned keys for directions, the cursor moves the initial step but no further. When a key is released, it resets to its original location. Assuming that this is likely down to how mouse capturing works?

**Comment by pdjstone** [comment]
Correct, this is due to the new mode handling which allows the mouse to be used without it being captured. For now you can work around the issue by forcing mouse capture, either by:
- Going full screen
- or adding `mouse-capture=force` to the URL, e.g. https://archi.medes.live/#preset=a5000&ff=15000&mouse-capture=force

There's not yet a way to change how the mouse is handled in the UI (it's on my todo list). I've been [considering a tweak](https://github.com/pdjstone/arculator-wasm/issues/17) to the heuristics that that should detect when something is trying to move the mouse cursor (such as Warfarin) and automatically switch to captured mode.
